src/vaillant_netatmo_api/thermostat.py

```diff
@@ -370,6 +436,12 @@ def __init__(
         self.system_mode = SystemMode(system_mode)
         self.setpoint_hwb = Setpoint(**setpoint_hwb)
         self.modules = [Module(**module) for module in modules]
+        self.home_id = home_id
+        self.boiler_id = boiler_id
+        self.dhw_always_on = dhw_always_on # this should be set later by entity update_method->_client.async_get_dhw_data
```

Comment:
`dhw_always_on` is not something `getthermostatsdata` API endpoint returns, which means it should not be a part of this model.

This is the [recommendation](https://developers.home-assistant.io/docs/api_lib_data_models) from Home Assistant which I'm following in this lib.
> When modelling the data, it is important that we expose the data from the API in the same structure as that the API offers it. Some API designs might not make a lot of sense or contain typos. It is important that we still represent them in our objects. This makes it easy for developers using your library to follow the API documentation and know how it will work in your library.

Reply:
So `dhw_always_on` will be part of other class

Reply:
Yep, the `ConfigModule` one.